packages/inter-protocol/src/proposals/econ-behaviors.js

```diff
@@ -40,6 +40,15 @@ const sanitizePathSegment = name => {
   return candidate;
 };
 
+/**
+ * There's an obsolete version in utils.d.ts
```

Comment:
please put these updates there instead so there's one correct one and no incorrect ones.
```ts
type AdminFacet = {
  getVatShutdownPromise: () => Promise<any>; // Completion, which is currently any
  upgradeContract: (contractBundleId: string, newPrivateArgs: any) => void;
  restartContract: (newPrivateArgs: any) => void;
};
```

Reply:
done